.all-contributorsrc

```diff
@@ -0,0 +1,44 @@
+{
+  "projectName": "typicalbot",
+  "projectOwner": "sylkellc",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 50,
+  "commit": false,
+  "commitConvention": "angular",
```

Comment:
This should be standard or conventional, as conventional commit doesn’t follow angular spec entirely.

Reply:
I was given neither as an option. Regardless, as I understand it, this setting has no effect as I'm the app isn't set to auto-commit.